lib/resourcemerge/os.go

```diff
@@ -51,7 +51,7 @@ func SetOperatorStatusCondition(conditions *[]configv1.ClusterOperatorStatusCond
 
 	if existingCondition.Status != newCondition.Status {
 		existingCondition.Status = newCondition.Status
-		existingCondition.LastTransitionTime = newCondition.LastTransitionTime
+		existingCondition.LastTransitionTime = metav1.NewTime(time.Now())
```

Comment:
`metav1.Now()`?

Reply:
https://github.com/openshift/cluster-version-operator/pull/153/files#diff-7b07a5cf0629a7b019ac699efb7cfe94R47

This is the same method used to generate time elsewhere in this file.

Reply:
yeah, just pointing out that there's a direct one, just a nit